rest-api/tools/install_config.py

```diff
@@ -56,9 +58,15 @@ def main(args):
     else:
       combined_config = config_file
     comparable_file = _json_to_sorted_string(combined_config)
+    # Don't compare 'aes_key' values because it is always changed when updating.
+    comparable_file = re.sub(r'\n.*?"aes_key".*?\n', '\n', comparable_file)
+    formatted_server_config = re.sub(r'\n.*?"aes_key".*?\n', '\n', formatted_server_config)
     configs_match = _compare_configs(comparable_file, formatted_server_config, args.config_output)
 
     if not configs_match and args.update:
+      # replace 'aes_key' with a new random 16-digit string.
+      new_key = ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(16))
+      combined_config['aes_key'] = new_key
```

Comment:
Isn't it going to be an issue when pagination suddenly breaks on every server release?

Reply:
I think I have an idea on how to change the key every 3 or 6 months without putting the key in the git project.